site/app/templates/admin/admin_gradeable/AdminGradeablePeers.twig

```diff
@@ -141,36 +169,157 @@
             contentType: false,
             success: function(data) {
                 $('.popup-form').css('display', 'none');
-                let new_peers = JSON.parse(data)["data"];
-                let grader_found = false;
-                for (grader in new_peers) {
-                    if (grader == grader_id) {
-                        grader_found = true;
-                    }
-                }
-                let grader_content = $('.peer-table-grader').filter(function() {
-                    return $(this).text() == grader_id
-                });
-                if (grader_found == false) {
-                    //the entire grader row is removed from new peers
-                    grader_content.parents('tr').remove();
-                } else {
-                    //otherwise, fill in new students in the table
-                    let students_list = grader_content.parents('tr').find('.peer-table-students');
-                    students_list.empty();
-                    for (i in new_peers[grader_id]) {
-                        students_list.append('<span>'+new_peers[grader_id][i]+'</span> ');
-                    }
-                }
+                window.location.reload(false);
```

Comment:
Can we ignore page refresh as we are already making an AJAX request and instead update the page with DOM manipulation?